plugins/tab-three/component.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+
+export default function TabThree({tabData}) {
+    return (
+        <div>
+            This is tab three.
+            <pre>
+                {JSON.stringify(tabData)}
+            </pre>
+        </div>
+    )
+}
```

Comment:
This could be anything, and can call its own redux flow. Since it will be tested and developed and compiled with the main app, it can directly reference parts of it, but we should explicitly mark APIs in that case.